tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -806,9 +809,60 @@ describe(__filename, () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
+  describe('enableInlineAddonReview', () => {
+    function renderInline(otherProps = {}) {
+      const _config = getFakeConfig({
+        enableInlineAddonReview: true,
+      });
+      return render({ _config, ...otherProps });
+    }
+
+    it('shows UserReview by default', () => {
+      const review = signInAndDispatchSavedReview();
+      const root = renderInline({ review });
+
+      expect(root.find(AddonReviewManager)).toHaveLength(0);
+      expect(root.find(UserReview)).toHaveLength(1);
+    });
+
+    it('renders AddonReviewManager when editing', () => {
+      const review = signInAndDispatchSavedReview();
+      store.dispatch(showEditReviewForm({ reviewId: review.id }));
+
+      const root = renderInline({ review });
+
+      expect(root.find(UserReview)).toHaveLength(0);
+      const manager = root.find(AddonReviewManager);
+      expect(manager).toHaveLength(1);
+      expect(manager).toHaveProp('review', review);
+    });
+
+    it('hides the review form on cancel', () => {
+      const review = signInAndDispatchSavedReview();
+      store.dispatch(showEditReviewForm({ reviewId: review.id }));
+      const dispatchSpy = sinon.spy(store, 'dispatch');
+
+      const root = renderInline({ review });
+
+      const manager = root.find(AddonReviewManager);
+      expect(manager).toHaveProp('onCancel');
+
+      dispatchSpy.resetHistory();
+      const onCancel = manager.prop('onCancel');
```

Comment:
Nit: this `const` seems a bit redundant.

Reply:
I don't have a strong opinion but, to me, `manager.prop('onCancel')()` is harder to read than `onCancel()`.